github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help-skipFirstRun.html

```diff
@@ -1,4 +1,7 @@
 <div>
-    If this checked, then first trigger run will not run real builds.
-    After first run this value automatically unset.
+    If selected, PRs that were pending before a first dry run of this job will
+    be ignored, and only PRs updated or created after that first dry run will
+    be built.
+    <br/>
+    NOTE: After selecting this option, you need to run the build once by hand.
```

Comment:
No any `by hand` builds are required. You can't even run by hand any PR build because publishers/builders rely on special Cause that injected only from programmatic run.